src/utils/WalletUtils.ts

```diff
@@ -142,3 +142,58 @@ export async function delegate(
         throw error
     }
 }
+
+
+
+export async function withdraw(
+    walletFileContents: string,
+    passphrase: string,
+    contractAddress: string,
+    amount: number,
+    tezosNodeAddress: string
+): Promise<string> {
+
+    const { privateKey, address } = await unlockWallet(walletFileContents, passphrase);
+
+    const tezos = new TezosToolkit(tezosNodeAddress);
+
+    tezos.setProvider({
+        signer: new InMemorySigner(privateKey)
+    });
+    const contract1 = await tezos.contract.at(contractAddress);
+    console.log({tezos})
+    console.log({contract1})
+
+    try {
+        // Withdraw the funds
+        const contract = await tezos.contract.at(contractAddress);
+        const op = await contract.methods.transfer({
```

Comment:
I don't think KT1 delegation contracts, i.e. manager.tz, contracts have a transfer() function:
- https://gitlab.com/nomadic-labs/mi-cho-coq/blob/master/src/contracts/manager.tz
- https://better-call.dev/mainnet/KT1GiNqN8zCeVfRL425Ts6stQYC8hAiQfSHb/code

As discussed on chat, here's a snippet to actually withdraw funds:

```typescript
const withdrawFromKT1 = (managerAddress: string, amountInMutez: string) => {
	return [
	   {
	      "prim":"DROP"
	   },
	   {
	      "prim":"NIL",
	      "args":[
	         {
	            "prim":"operation"
	         }
	      ]
	   },
	   {
	      "prim":"PUSH",
	      "args":[
	         {
	            "prim":"key_hash"
	         },
	         {
	            "string": managerAddress
	         }
	      ]
	   },
	   {
	      "prim":"IMPLICIT_ACCOUNT"
	   },
	   {
	      "prim":"PUSH",
	      "args":[
	         {
	            "prim": "mutez"
	         },
	         {
	            "int": amountInMutez
	         }
	      ]
	   },
	   {
	      "prim":"UNIT"
	   },
	   {
	      "prim":"TRANSFER_TOKENS"
	   },
	   {
	      "prim":"CONS"
	   }
	]
}

const contract = await Tezos.contract.at('kt1...');
await contract.methodsObject.do(withdrawFromKT1('tz1..', 1234)).send();
```

This is adapted from the code in the links above and https://taquito.io/docs/set_delegate/#delegation-involving-originated-addresses-kt1-prefix.